pkg/brctl/brctl.go

```diff
@@ -0,0 +1,340 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package brctl
+
+import (
+	"encoding/binary"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path"
+	"strconv"
+	"strings"
+
+	"golang.org/x/sys/unix"
+)
+
+// Addbr adds a bridge with the provided name.
+func Addbr(name string) error {
+	brctlSocket, err := unix.Socket(unix.AF_INET, unix.SOCK_STREAM, 0)
+	if err != nil {
+		return fmt.Errorf("unix.Socket: %w", err)
+	}
+
+	if _, err := executeIoctlStr(brctlSocket, unix.SIOCBRADDBR, name); err != nil {
+		return fmt.Errorf("executeIoctlStr: %w", err)
+	}
+
+	return nil
+}
+
+// Delbr deletes a bridge with the name provided.
+func Delbr(name string) error {
+	brctl_socket, err := unix.Socket(unix.AF_INET, unix.SOCK_STREAM, 0)
```

Comment:
I missed some names from previous review. Please avoid underscores in names if possible.